.github/workflows/release-pypi.yml

```diff
@@ -9,46 +9,164 @@ on:  # Trigger the workflow on push or pull request, but only for the master bra
 
 
 jobs:
-  # Todo run job which determine changed versions
+  # run job which determine changed versions
```

Comment:
Can you add a lot more comments to those new CI? This would be hard for anyone else to get through them. Better to invest in comments right now while it is fresh.

Reply:
there is not much new, just two jobs or so, but the titles are quite descriptive and added the top-level description/flow